dataservice/api/__init__.py

```diff
@@ -12,6 +15,25 @@
           default_label='')
 
 api.add_namespace(person_api)
+api.add_namespace(demographic_api)
+
+
+@api.errorhandler
+def default_error_handler(error):
+    """
+    Default error handler
+    """
+    code = getattr(error, 'code', 500)
+    return kf_response(code=code, message=str(error))
+
+
+# TODO
+"""
```

Comment:
As a work around we may need to change our kf_response format a little bit to match the default restplus format. So instead of having 
```
    {
      "_links": {
        ...
      },
      "_status": {
        "code": 200,
        "message": "Success"
      },
      "results": [
        ...
      ]
    }
    ```
We might need to change to something like: 
```
    {
      "_links": {
        ...
      },
      "code": 200,
      "message": "Success"
      "results": [
        ...
      ]
    }
    ```